.github/integration_tests/compliant_repo.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Integration test to ensure CI fails when a non-existent repository is added
+
+# set git credentials
+git config user.name dfinity
+git config user.email dfinity@dfinity.org
+
+# create new branch
+git checkout -b integration-test-2-$RUN_NUMBER
```

Comment:
Some meaningful name instead of just `integration-test-1` and `integration-test-2`?

Reply:
I don't know that it really matters since the branch gets deleted. The PR description and CI description have both been updated. I find it helpful to have a distinction between regular branches and integration test branches.